contrib/IECoreMantra/src/IECoreMantra/ProceduralPrimitive.cpp

```diff
@@ -179,7 +184,8 @@ void IECoreMantra::ProceduralPrimitive::addVisibleRenderable( VisibleRenderableP
 			addGeometry(gdp, 0.0f);
 			while ( !m_renderer->m_motionTimes.empty() )
 			{
-				setPreTransform( convert< UT_Matrix4T<float> >(m_renderer->m_motionTransforms.front()),
```

Comment:
H15 does not require the float matrix.

I'm not sure, I get a problem without converting to double.
I tried this:
`
				UT_Matrix4T<double> frontTransform =  convert< UT_Matrix4T<double> >( m_renderer->m_motionTransforms.front() );
				setPreTransform( frontTransform ,
								 m_renderer->m_motionTimes.front() );
`

and got this result: 

`include/IECore/Convert.h: In instantiation of 'T IECore::convert(const F&) [with T = UT_Matrix4T<double>; F = Imath_2_2::Matrix44<float>]':
contrib/IECoreMantra/src/IECoreMantra/ProceduralPrimitive.cpp:187:114:   required from here
include/IECore/Convert.h:49:9: error: could not convert 'from' from 'const Imath_2_2::Matrix44<float>' to 'UT_Matrix4T<double>'
  return from;
         ^
include/IECore/Convert.h: In function 'T IECore::convert(const F&) [with T = UT_Matrix4T<double>; F = Imath_2_2::Matrix44<float>]':
include/IECore/Convert.h:50:1: error: control reaches end of non-void function [-Werror=return-type]
 }
`